### ISSUE TITLE
[Request] Instant ground target on button-up instead of button-down

### ISSUE THREAD

**Issue by jmpnul** [UnknownX7/ReAction#133]
I play on controller, and cannot see my cursor when using the instant ground target feature. I'm worried that my cursor has moved and I'll place the ability in the wrong spot. I'm not sure if it's possible to implement this for controller players, but it would be nice if press-and-holding an ability displayed a reticle (locked to the centre of the screen), and releasing the button executed the ability instantly.

In short:
1. Button down event
2. Lock cursor to centre of screen (or a custom defined position)
3. Overlay a cursor and/or reticle to help make micro adjustments
4. Button up event triggers the ability
